Source/EasyNetQ/Consumer/DefaultConsumerErrorStrategy.cs

```diff
@@ -41,7 +41,7 @@ public class DefaultConsumerErrorStrategy : IConsumerErrorStrategy
         ///     Creates DefaultConsumerErrorStrategy
         /// </summary>
         public DefaultConsumerErrorStrategy(
-            IPersistentConnection connection,
+            IConsumerConnection connection,
```

Comment:
It is a vital here to have `IConsumerConnection`, because `IProducerConnection` could be blocked because of publishes and this fact will yield to hanged consumers.